BotModules/Megabot/index.js

```diff
@@ -1,9 +1,14 @@
 const Discord = require('discord.js');
 const bot = new Discord.Client();
 
+const Commander = require('../CommanderInChief/commander-in-chief');
+
+Commander.attachToBot(bot);
```

Comment:
Why are we attaching the bot like this? the rest of the bots attach at the constructor.